File: app/src/main/java/com/uneatlantico/uneapp/db/UneAppDB.java
package com.uneatlantico.uneapp.db;

import android.content.Context;

import com.readystatesoftware.sqliteasset.SQLiteAssetHelper;

public class UneAppDB extends SQLiteAssetHelper {

    private static final String DATABASE_NAME = "UneAppDatabase.db";
    private static final int DATABASE_VERSION = 1;

    public UneAppDB(Context context) {
        super(context, DATABASE_NAME, "data/data/com.uneatlantico.uneapp/databases/", null, DATABASE_VERSION);
    }

}

File: app/src/main/java/com/uneatlantico/uneapp/db/UneAppDB.kt
package com.uneatlantico.uneapp.db

import com.orm.SugarRecord

class UneAppDB: SugarRecord {
    var idEvento: Long
    var fecha: String

    constructor(idEvento: Long, fecha: String){
        this.idEvento = idEvento
        this.fecha = fecha
    }
}